pkg/cmd/server/origin/run_components.go

```diff
@@ -162,8 +162,7 @@ func (c *MasterConfig) RunDNSServer() {
 	}
 
 	if ok, err := cmdutil.TryListen(c.Options.DNSConfig.BindNetwork, c.Options.DNSConfig.BindAddress); !ok {
-		glog.Warningf("Could not start DNS: %v", err)
-		return
+		glog.Fatalf("Could not start DNS: %v", err)
```

Comment:
It is, see the issue.  Needs to be a bit more nuanced.  I need to sit down
and talk out with folks the things Jordan and I were riffing on about
changing the start flow

On Dec 4, 2015, at 10:29 AM, Jordan Liggitt <notifications@github.com>
wrote:

In pkg/cmd/server/origin/run_components.go
<https://github.com/openshift/origin/pull/6195#discussion_r46693074>:

> @@ -162,8 +162,7 @@ func (c *MasterConfig) RunDNSServer() {
>  	}
>
>  	if ok, err := cmdutil.TryListen(c.Options.DNSConfig.BindNetwork, c.Options.DNSConfig.BindAddress); !ok {
> -		glog.Warningf("Could not start DNS: %v", err)
> -		return
> +		glog.Fatalf("Could not start DNS: %v", err)

hmm, @smarterclayton <https://github.com/smarterclayton> might have
thoughts on this... pretty sure this was intentional

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/6195/files#r46693074>.